src/uu/stty/src/stty.rs

```diff
@@ -478,6 +486,29 @@ fn parse_rows_cols(arg: &str) -> Option<u16> {
     None
 }
 
+fn parse_saved_state(arg: &str) -> Option<Vec<u32>> {
```

Comment:
please add comments 
and it would be nice to have unit tests with valid and invalid data here

Reply:
Good call on the testing, I added the functionality to compare a wide range of inputs and outputs against the GNU implementation and found that the error messaging was not matching and that the failure messages had the help message included in the error messages.